### DISCUSSION TITLE
The majority of results in 'clones_TR[A|B].tsv' are "region_not_covered"

### DISCUSSION THREAD

**Discussion by ChoiJi-Hye** [milaboratory/mixcr#1688] [Q&A] [answered]
Hi! 
Thanks for creating a great tool!

## Expected Result

I used [TakaRa] SMART-Seq Human TCR (with UMIs) to generate bulk TCR data and used your tool for analysis, but most of the results came out as "region_not_covered." 
When I look at other people's results, most of them seem to have sequences written. Is there an option I might have missed?

I proceeded with the analysis using **takara-human-rna-tcr-umi-smartseq**, but is this an accurate result? 
Are there any other issues?

## Actual Result

![image](https://github.com/milaboratory/mixcr/assets/43129945/f7c00b74-a88d-42f3-9c79-27d87a420e71)

## Exact MiXCR commands 

`Mixcr_4.6.0/mixcr analyze takara-human-rna-tcr-umi-smartseq R_23_00645_FN_ETC_RNA_R1.fastq.gz R_23_00645_FN_ETC_RNA_R2.fastq.gz result_directory/R_23_00645 -t 16` 

## MiXCR report files 

![image](https://github.com/milaboratory/mixcr/assets/43129945/ae09b9b3-3d79-4c50-8831-60dcee8947ce)

Thank you for your help!
Ji-Hye Choi

**Comment by mizraelson** [comment]
The command seems correct. It appears that the issue is with the data itself. Could you please share the pair of raw FASTQ files? I can then review them and provide some insights. You can reach us at [support@milaboratories.com](mailto:support@milaboratories.com)

**Reply by mizraelson** [reply, marked as answer]
Thank you for sharing the data. It appears that there is an issue with the read length. By default, the preset expects sequencing of 300+300, in accordance with the manufacturer's recommendations. In your case, with reads of 150+150, the reads do not fully cover the VDJ region. Please add the parameter `--assemble-clonotypes-by CDR3` to assemble clones by the CDR3 region instead of by the full-length sequence.

**Reply by ChoiJi-Hye** [reply]
Thank you so much for your help! 

Following your advice, I was able to increase the read count in the result files and obtain both the nucleotide and amino acid sequences for the CDR3 region. 

<img width="1298" alt="image" src="https://github.com/milaboratory/mixcr/assets/43129945/480b11d0-bb61-4394-8891-e88f704f51dc">


I really appreciate your quick assistance!
